docker-compose/docker-compose-extras.yaml

```diff
@@ -14,7 +14,7 @@ services:
       - ANSYS_LOCK="OFF"
   pydpf-composites-server:
     restart: unless-stopped
-    image: ${IMAGE_NAME_DPF_COMPOSITES:-ghcr.io/ansys/pydpf-composites:2023r2_pre1}
+    image: ${IMAGE_NAME_DPF_COMPOSITES:-ghcr.io/ansys/pydpf-composites:latest}
```

Comment:
If this turns out to be too unstable, we can switch back to the last explicitly tagged version `2025r1`.